### PR TITLE
Md sc1 65 19

### PR DESCRIPTION
**Pull request by viktarradzin**
Home work

### CODE REVIEW

File: vradzin/05_HTML/task_01.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>  
+<HTML> 
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Cпрайты</TITLE> 
+        <style type="text/css">
+        .s_1 {
```
Comment (pluhin) [inline vradzin/05_HTML/task_01.html]: Could you please minimize that CSS

**Review by pluhin — CHANGES_REQUESTED** [review]

File: vradzin/05_HTML/task_01.html (hunk 1 above)
Comment (pluhin) [reply vradzin/05_HTML/task_01.html]: Still pending

**Review by pluhin — CHANGES_REQUESTED** [review]

File: vradzin/05_HTML/task_01.html (hunk 1 above)
Comment (viktarradzin) [reply vradzin/05_HTML/task_01.html]: add new code

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**